CITATION.cff

```diff
@@ -0,0 +1,18 @@
+# YAML 1.2
+---
+cff-version: "1.2.0"
+title: "mibipret"
+authors:
+  -
+    family-names: Zech
+    given-names: Alraune
+    orcid: "https://orcid.org/0000-0000-0000-0000"
```

Comment:
Maybe you can put your ORCID here. That will make it easier for people to find you.

Reply:
```suggestion
    orcid: "https://orcid.org/0000-0002-8783-6198"
```